src/commands/openInRemoteSsh.ts

```diff
@@ -0,0 +1,50 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fse from 'fs-extra';
+import { join } from 'path';
+import * as SSHConfig from 'ssh-config';
+import { commands } from 'vscode';
+import { IActionContext } from 'vscode-azureextensionui';
+import { ext } from '../extensionVariables';
+import { localize } from '../localize';
+import { VirtualMachineTreeItem } from '../tree/VirtualMachineTreeItem';
+import { sshFsPath } from '../utils/sshUtils';
+import { addSshKey } from './addSshKey';
+import { verifyRemoteSshExtension } from './verifyRemoteSshExtension';
+
+export async function openInRemoteSsh(context: IActionContext, node?: VirtualMachineTreeItem): Promise<void> {
+    if (!node) {
```

Comment:
Should you verify `node` has the `VirtualMachineTreeItem.linuxContextValue` context value here just to be safe?

Reply:
Hmm, I don't think it's necessary since we're doing that check in the `package.json`.